deps/rabbit/src/rabbit_deprecated_feature_extra.erl

```diff
@@ -0,0 +1,62 @@
+%% This Source Code Form is subject to the terms of the Mozilla Public
+%% License, v. 2.0. If a copy of the MPL was not distributed with this
+%% file, You can obtain one at https://mozilla.org/MPL/2.0/.
+%%
+%% @copyright 2018-2023 VMware, Inc. or its affiliates.
+%%
+%% @doc
+%% This module provides extra functions unused by the feature flags
+%% subsystem core functionality.
+
+-module(rabbit_deprecated_feature_extra).
```

Comment:
I would prefer a module name slightly more obscure to convey the idea that it's internal to the deprecated features subsystem and not an API entrypoint for the rest of RabbitMQ. Something like `rabbit_depr_ff_extra`.